posthog/hogql/test/test_hogql.py

```diff
@@ -61,7 +61,9 @@ def test_hogql_materialized_fields_and_properties(self):
 
     def test_hogql_methods(self):
         self.assertEqual(self._translate("count()"), "count(*)")
-        self.assertEqual(self._translate("count(event)"), "count(distinct event)")
+        self.assertEqual(self._translate("countDistinct(event)"), "count(distinct event)")
```

Comment:
What happens if one does `count(properties.value == 2)`?

Reply:
Error because it expects just 0 arguments. I'd add the test, but the tests pass right now so it'll come in another PR 😅

Reply:
Oh yeah, sorry, I meant `countDistinct`

Reply:
Well, give it a [try](https://app.posthog.com/events#q=%7B%22kind%22%3A%22DataTableNode%22%2C%22full%22%3Atrue%2C%22source%22%3A%7B%22kind%22%3A%22EventsQuery%22%2C%22select%22%3A%5B%22event%22%2C%22countDistinct(properties.%24browser%20%3D%3D%20'Chrome')%22%2C%22countDistinct(toInt(properties.foo)%20%3D%3D%202)%22%5D%2C%22after%22%3A%22-24h%22%2C%22limit%22%3A100%7D%2C%22propertiesViaUrl%22%3Atrue%7D) :) 

I guess the answer is you'll get a `0`, `1` or `2` depending on the data.